frontend/src/controller/api.ts

```diff
@@ -4,7 +4,7 @@ import { Member } from '../../../src/typedefs/member';
 export default async function me(token: string): Promise<Member> {
     const response = await fetch(`${process.env.REACT_APP_API_URL}/api/me`, {
         method: 'GET',
-        mode: 'no-cors',
+        mode: 'cors',
```

Comment:
We may need to test this further - I recall not having `no-cors` specified leading to problems

Reply:
From what I can remember from my reading on Sunday, the 'no-cors' mode makes all requests and responses opaque. This prevents the use of the Authorization header, which is why it wasn't showing up when we were debugging